### DISCUSSION TITLE
PytestAssertRewriteWarning: asserting the value None, please use "assert is None"

### DISCUSSION THREAD

**Discussion by Vaish-09** [pytest-dev/pytest#8120] [Q&A] [answered]
I was trying to solve an error regarding assert none when working on a hands-on. This query was already raised, but even after following the steps, the issue still persists. The discussion was closed and hence [https://github.com/pytest-dev/pytest/issues/6203](url) and hence I opened a new one.
My code below works in the preview area, but during the Test Case run, the following error is displayed:

test/test_webpage.py::TestWebpage::test_indexpage

/projects/challenge/test/test_webpage.py:27: PytestAssertRewriteWarning: asserting the value None, please use "assert is None"

assert self._index.find('video', {'autoplay': ''})

The code already has the autoplay attribute set like how it is expected in the form of key value pairs. But still I am getting an error as above. Code is attached as markdown.

![videohtmlcode](https://user-images.githubusercontent.com/56498492/101841826-bf7bcb00-3b6c-11eb-8ca9-31d84d39c44e.PNG)
Can someone please help me out?

**Comment by asottile** [comment, marked as answer]
This warning was misleading and so it was removed in pytest 5.3.2 -- I'd suggest upgrading pytest
